src/index.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__content">
+    <ul class="calendar calendar--start-day--sun calendar--month-length--31">
+      <li class="calendar__cell calendar__cell--1"></li>
```

Comment:
let's get rid of extra class with added number of the `li` element and will use `nth-child` instead of extra class